engine/core/src/main/java/org/datacleaner/job/builder/ComponentBuilderTransformerChangeListener.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.job.builder;
+
+import java.util.List;
+
+import org.datacleaner.api.InputColumn;
+import org.datacleaner.data.MutableInputColumn;
+import org.datacleaner.descriptors.ConfiguredPropertyDescriptor;
+import org.datacleaner.util.ReflectionUtils;
+
+public class ComponentBuilderTransformerChangeListener implements TransformerChangeListener {
+    private final ComponentBuilder _componentBuilder;
+    private final ConfiguredPropertyDescriptor _propertyDescriptor;
+
+    public ComponentBuilderTransformerChangeListener(ComponentBuilder componentBuilder,
+            ConfiguredPropertyDescriptor propertyDescriptor) {
+        _componentBuilder = componentBuilder;
+        _propertyDescriptor = propertyDescriptor;
+    }
+
+    @Override
+    public void onAdd(TransformerComponentBuilder<?> builder) {
+        // Do nothing.
```

Comment:
I've seen a lot of this in other implementations of TransformerChangeListener... Should we make an adapter at some point (not now)?